.changeset/dirty-grapes-vanish.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-defaults': patch
+---
+
+The `DefaultSchedulerService` now accepts `HttpRouterService` and `PluginMetadataService` arguments. If you supply a router, the scheduler will register a REST API for listing and triggering tasks.
```

Comment:
I think this also deserves an explanation of the new feature 😅

Reply:
An expansion of the "REST API for listing and triggering tasks" to be more descriptive? Sure, I can add that

Reply:
Added complete docs to https://backstage.io/docs/backend-system/core-services/scheduler

Reply:
Noice 👌